### DISCUSSION TITLE
disableIdentityMap

### DISCUSSION THREAD

**Discussion by lineCode** [mikro-orm/mikro-orm#5290] [General]
how to set disableIdentityMap when use QueryBuilder

**Comment by B4nan** [comment]
Its a feature of EM, not QB. To mimic what it does on QB level, just fork first before you create it.

**Reply by lineCode** [reply]
i found em.find options has disableIdentityMap，but how to set when use qb, fork options can not found this

**Reply by lineCode** [reply]
use em.config.set('disableIdentityMap', true)?, then fork, and create qb?

**Reply by B4nan** [reply]
there are no options, i told you to fork first before you create the QB, not to use some option

```ts
const qb = em.fork().createQueryBuilder();
// ...
```

thats the equivalent
